projects/cn0561/builds.json

```diff
@@ -0,0 +1,12 @@
+{
+  "xilinx": {
```

Comment:
we might need xsa files for this to work with the projects build

Reply:
Will provide.